main/src/main/scala/sbt/Script.scala

```diff
@@ -32,7 +42,7 @@ object Script {
       val append = Load.transformSettings(Load.projectScope(currentRef), currentRef.build, rootProject, scriptSettings ++ embeddedSettings)
 
       val newStructure = Load.reapply(session.original ++ append, structure)
-      val arguments = state.remainingCommands.drop(1)
+      val arguments = state.remainingCommands.drop(1).map(arg => "\"" + arg + "\"")
```

Comment:
This is on the windows command line. I was just trying different scenarios to get sbt to print an argument with a space. The `run` above is the same code as the sbt script example above. You can try seeing if you can run a script like above `hellosbt.scala` and try passing "sbt World". It should ignore the "World".